components/dashboard/sections/overview/queries.ts

```diff
@@ -98,17 +101,25 @@ export const timelineQuery = gql`
           summary
           slug
         }
+        conversation {
+          id
+          title
+          summary
+          slug
+        }
         individual {
           id
           slug
           name
           type
-          imageUrl(height: 48)
+          imageUrl
```

Comment:
We're removing it from here because the Avatar component already adds the required height, correct?
I guess that, for caching reasons, we should use a single maximum required size image and just have the browser resize it.

Reply:
Removed it from here since it interfered with adding the AccountHoverCardFields fragment that has imageUrl without the size, but yes, the Avatar component will create a URL to the image server asking for the required size.

Sounds like a nice enhancement of the Avatar component to ask for a single image!